README.md

```diff
@@ -54,6 +54,14 @@ Specifically, we introduce the following changes inspired by our experiments and
 
 1. **Multiple cosmetic/minor changes**: Change in robot embodiment/size, camera resolution, height, and orientation, etc &mdash; to match LoCoBot. 
 
+1. **Notes about performance**: We have noticed that turning on the RGB/Depth sensor noise may lead to worse FPS performance. As such, we recommend training with these noises turned off and using them for fine tuning if necessary. This can be down by commenting out the lines that include the key "noise_model" in the config or by overriding these values from the command line.
```

Comment:
```suggestion
1. **Notes about performance**: We have noticed that turning on the RGB/Depth sensor noise may lead to reduced simulation speed. As such, we recommend initially training with these noises turned off and using them for fine tuning if necessary. This can be done by commenting out the lines that include the key "noise_model" in the config or by overriding these values from the command line.
```